### ISSUE TITLE
Not listing private repositories for an organization I'm a member of

### ISSUE THREAD

**Issue by mikebroberts** [ccmenu/ccmenu2#9]
I've successfully authenticated a GitHub user, and am able to add both the public-repo workflow of an organization, and the private-repo workflow of the user itself. However when I enter the name of an organization that the user is a member of in the "Owner" box it doesn't list the private repositories in that organization. If I visit the organization in the GitHub Web App with the same user I can see both the private repositories, and the workflows.

I suspect the problem here is that in _GitHubAPI.swift_ to list the repositories you're either doing:

* `/users/{org}/repos` for the public repos of an owner, or
* `/user/repos` for the private **personal** repos of the user

But I _think_ for private repos of an organization you need to call `/orgs/{org}/repos` (note it's `orgs` not `users` in the first level) (see https://docs.github.com/en/enterprise-server@3.9/rest/repos/repos?apiVersion=2022-11-28)

**Comment by erikdoe** [comment]
Yes and no. CCMenu calls both endpoints. You can see the logic [here](https://github.com/ccmenu/ccmenu2/blob/27145f411a49f928f6251c42a041bb582f2246a6/CCMenu/Source/Pipeline%20Window/GitHub%20Sheets/GitHubRepositoryList.swift#L20-L43). But it looks like CCMenu needs to make a third API call to get the private org repos.

**Comment by erikdoe** [comment]
In the end the solution was slightly different: first find out whether the _owner_ string is a user or an org, with another API call, and then make the correct calls.

**Comment by erikdoe** [comment]
Please have a look whether this pre-release addresses your issue: https://github.com/ccmenu/ccmenu2/releases/tag/v27.0d1

**Comment by mikebroberts** [comment]
Hi @erikdoe - unfortunately it doesn't seem to. I made sure to trigger a GitHub action workflow in a private repo, but I still don't see the repo in the "Repository" list for my organization.

**Comment by mikebroberts** [comment]
Hi @erikdoe - Sorry for the delay, I was waiting to push to the main branch of one my org+private repos.

So I figured out one problem - I hadn't "granted" permission to the CCTray OAuth App in the Github Login / Auth dialog. Once I did that I can see my private workflows in the Add Pipeline window.  However now I have a new problem - when I press 'Apply' in that dialog.

I just installed 27.0 from the releases page before trying.
